src/routing.js

```diff
@@ -163,6 +163,20 @@ export function LocationProvider({
   } />
 }
 
+export function StaticLocationProvider({ location, children }) {
```

Comment:
Ik zie geen update in de README? Is denk ik wel fijn om deze even te updaten met een kort stukje over hoe je deze provider kunt gebruiken om page transities in je applicatie toe te voegen